app/routes/questionnaire.py

```diff
@@ -385,7 +384,7 @@ def get_view_submitted_response(schema, questionnaire_store):
             flask_babel.get_locale().language,
         )
 
-    except (ViewSubmittedResponseNotEnabled, ViewSubmittedResponseExpired):
+    except (ViewSubmittedResponseNotEnabled):
```

Comment:
The parenthesis is now redundant.
```suggestion
    except ViewSubmittedResponseNotEnabled:
```